### PR TITLE
release 6.1.0

### PR DESCRIPTION
**Pull request by gmmeyer**
### What does this PR do?

Releases 6.1.0!

### CODE REVIEW

File: release.json [hunk 1]
```diff
@@ -6,6 +6,13 @@
 		"JMXFETCH_VERSION": "0.19.0",
 		"JMXFETCH_HASH": "28f345c2d407e85802c43201e1e1ea5b8664f39ac705ac8e6b44295b641c5d70"
 	},
+    "6.1.0": {
```
Comment (olivielpeau) [inline release.json]: weird indentation here

**Review by olivielpeau — DISMISSED** [review]
one nit, other than that LGTM 👍

File: release.json (hunk 1 above)
Comment (gmmeyer) [reply release.json]: for some reason this file is tabs

Comment (gmmeyer) [reply release.json]: this was not tab. I made it tab

**Review by olivielpeau — APPROVED** [review]
:shipit:

**Pull request merged**